addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -20,7 +20,7 @@ This binding can discover the supported Onkyo AV Receivers. At the moment only t
 -   TX-NR525
 -   TX-NR535
 -   TX-NR555
--   TX-NR535
+-   TX-NR575E
```

Comment:
Why remove the `TX-NR535`? Is it a duplicate?

Reply:
Yes, please see two lines above.